inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/constant/FileConstants.java

```diff
@@ -13,16 +13,22 @@
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
+ *
  */
 
 package org.apache.inlong.agent.constant;
 
 /**
- * Collection type of file data
+ * file constants
  */
-public class FileCollectType {
+public enum FileConstants {
 
-    public static final String INCREMENT = "INCREMENT";
+    //file collect type
```

Comment:
it's better to keep the `FileCollectType` file name and create another file for file content type.

Reply:
done